pyment/docstring.py

```diff
@@ -1284,7 +1284,8 @@ class DocString(object):
     """This class represents the docstring"""
 
     def __init__(self, elem_raw, spaces='', docs_raw=None, quotes="'''", input_style=None, output_style=None,
-                 first_line=False, trailing_space=True, type_stub=False, before_lim='', **kwargs):
+                 first_line=False, trailing_space=True, type_stub=False, before_lim='', num_of_spaces=4,
```

Comment:
Thanks for putting in the work to make the indentation a variable with a default value.

Reply:
yep, no problem!